.dev.Makefile

```diff
@@ -18,7 +18,7 @@ DOCKER_BUILDER_ARGS = --rm -ti -v "$(realpath .):/app"
 
 SPECULOS_TAG=latest
 SPECULOS_IMAGE = ghcr.io/ledgerhq/speculos:$(BUILDER_TAG)
-SPECULOS_ARGS = --sdk 2.0 --display headless
+SPECULOS_ARGS = --display headless
```

Comment:
SDK 2.0 only works for nano S, leaving out the sdk makes speculos use the latest sdk for the device